apps/studio/pages/project/[ref]/functions/secrets.tsx

```diff
@@ -3,14 +3,16 @@ import { FunctionsSecretsEmptyStateLocal } from 'components/interfaces/Functions
 import DefaultLayout from 'components/layouts/DefaultLayout'
 import EdgeFunctionsLayout from 'components/layouts/EdgeFunctionsLayout/EdgeFunctionsLayout'
 import { PageLayout } from 'components/layouts/PageLayout/PageLayout'
-import { ScaffoldContainer } from 'components/layouts/Scaffold'
+import { ScaffoldContainer, ScaffoldSection } from 'components/layouts/Scaffold'
 import { IS_PLATFORM } from 'lib/constants'
 import type { NextPageWithLayout } from 'types'
 
 const SecretsPage: NextPageWithLayout = () => {
   return (
-    <ScaffoldContainer size="large" className="pt-6">
-      {IS_PLATFORM ? <EdgeFunctionSecrets /> : <FunctionsSecretsEmptyStateLocal />}
+    <ScaffoldContainer size="large">
```

Comment:
I assume this `size="large"` is intentionally different to other (skinnier) settings pages, like Storage Settings and Storage Policies, given that this one deals with tabular data.